events/2017/12-contributor-summit/ContribSummitInformation.md

```diff
@@ -18,7 +18,7 @@ Times in CST
 - 01:00 pm - Sessions resume  
 - 05:00 pm - Happy Hour onsite  
 
-View the complete session schedule [here](/community/2017-events/12-contributor-summit/schedule.png).  
+View the complete session schedule [here](/events/2017/12-contributor-summit/schedule.png).  
```

Comment:
If it's in the same folder, perhaps we just link this to `schedule.png` and then it'll be correct wherever this moves.

Reply:
Ah of course.  Amended.